llama-index-core/llama_index/core/vector_stores/types.py

```diff
@@ -93,12 +93,7 @@ class MetadataFilter(BaseModel):
     """
 
     key: str
-    value: Union[
-        StrictInt,
-        StrictFloat,
-        StrictStr,
-        List[Union[StrictInt, StrictFloat, StrictStr]],
-    ]
+    value: Union[StrictInt, StrictFloat, StrictStr, List[StrictStr]]
```

Comment:
Why did this type change?

Reply:
This is to allow array data as the filter value

Reply:
@brenkehoe Sorry for replying in a merged PR. Why is the support for `List[StrictInt]` and `List[StrictFloat]` being removed here?